src/libawkward/array/ListOffsetArray.cpp

```diff
@@ -242,7 +242,7 @@ namespace awkward {
   ListOffsetArrayOf<T>::compact_offsets64(bool start_at_zero) const {
     int64_t len = offsets_.length() - 1;
     Index64 out(len + 1);
-    struct Error err = util::awkward_listoffsetarray_compact_offsets64<T>(
+    struct Error err = kernel::listoffsetarray_compact_offsets64<T>(
```

Comment:
Similar to above, both the kernel dispatch and underlying kernel need an underscore in their names: `..._compact_offsets_64`.